mlflow/docker-compose.yaml

```
@@ -0,0 +1,12 @@
+services:
+  mlflow:
+    build: .
+    image: ghcr.io/mlflow/mlflow
+    ports:
+      - 5000:5000
+    environment:
+      MLFLOW_SERVER_DEFAULT_ARTIFACT_ROOT: /mlflow/artifacts
+    command: mlflow server --backend-store-uri postgresql://postgres:password@db:5432/mlflow --host 0.0.0.0
```

Add `POSTGRES_PORT` variable to the `.env` file.
```suggestion
    command: mlflow server --backend-store-uri postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:${POSTGRES_PORT}/mlflow --host 0.0.0.0
```